components/bigcommerce/actions/common/common.mjs

```diff
@@ -0,0 +1,486 @@
+const PROPS = {
```

Comment:
Hey, Looks like you are using this file only in `create-product`. I did not see this being used in any other, please. Looks like you did this to avoid putting all these props in that file. If this is the case, I would suggest you to rename this file to `create-product-props.mjs` or something similar and export it as default directly. Maybe this file can be moved to a `utils` folder or something similar.